### ISSUE TITLE
[FIXED] Reported version (0.1.0) is different than what is possible to see in the other places (5.2.1)

### ISSUE THREAD

**Issue by xambroz** [radareorg/iaito#34]
## Environment Details

Despite we only fix issues reproducible in git-master branch, it's fine to report issues found in releases. And those will be closed when a new release comes out.

```sh
# copypaste this script into your shell and replace it with the output
iaito -v
date
r2 -v
uname -ms
```

$ iaito -v
QCommandLineParser: already having an option named "w"
iaito 0.1.0
$ date
2021-04-16T22:14:58 CEST
$ r2 -v
radare2 5.2.0 0 @ linux-x86-64 git.5.2.0
commit: unknown build: 2021-04-16__00:49:31
$ uname -ms
Linux x86_64

## Description

iaito -v 
QCommandLineParser: already having an option named "w"
iaito 0.1.0

iaito / About / Version 0.1.0

**Comment by xambroz** [comment]
![Screenshot from 2021-04-16 22-16-43](https://user-images.githubusercontent.com/723625/115079546-a364ad80-9f01-11eb-9927-6cc537104b66.png)
![Screenshot from 2021-04-16 22-18-41](https://user-images.githubusercontent.com/723625/115079620-be372200-9f01-11eb-91b4-3580a264be65.png)

**Comment by trufae** [comment]
This is fixed in git. The fix will be available for 5.2.1
